Translate "Weekly Update - December 2nd, 2016" (#471)

devjin0617 opened the pull request.

devjin0617 committed:
Translate "Weekly Update - December 2nd, 2016" (#471)

outsideris commented on source/_posts/2016-12-02-weekly.md:
```diff
@@ -0,0 +1,90 @@
+The Node.js Foundation announced progress with efforts to make Node.js VM-neutral -- more information on this news can be found on the Node.js Foundation Medium blog. This morning, the Foundation announced it would oversee the Node.js Security Project to further improve stability for enterprises. More information here.
+-->
+
+Node.js 재단은 Node.js VM 중립을 만들기위한 노력으로 성과를 발표했습니다. 이 뉴스에 대한 자세한 내용은 Node.js Foundation Medium 블로그에서 확인할 수 있습니다. 금일 오전, 기업의 안정성을 더욱 향상시키기 위해 Node.js 보안 프로젝트를 관리할 것이라고 발표했습니다. 자세한 내용은 여기를 참조하세요.
```
`만들기위한` -> `만들기 위한`

`Noe.js를 VM 중립적으로 만들기 위한`으로 풀어쓰는 건 어떨까요?

Node.js Foundation -> Node.js 재단

devjin0617 replied:
수정하였습니다. 감사합니다.

outsideris commented on source/_posts/2016-12-02-weekly.md:
```diff
@@ -0,0 +1,90 @@
+* We are doing pretty well at 89.37% JS coverage and 88.3% C++ coverage but we can always do better.
+-->
+
+* Node.js 마스터(브랜치)는 https://coverage.nodejs.org/에서 확인 가능합니다. 당신의 주요 풀 리퀘스트가 적용된 후 커버리지가 낮아지지 않았는지 결과를 확인해주세요.
```
링크를 `<>`로 묶어주세요. 조사때문에 링크가 깨질것 같습니다.

devjin0617 replied:
링크부분을 찾아서 처리한다는 것을 놓쳐버렸네요. 수정하였습니다. 감사합니다.

devjin0617 committed:
Translate "Weekly Update - December 2nd, 2016" (#471)

-`만들기위한 → 만들기 위한
- Node.js VM 중립을 → Node.js를 VM 중립적으로
- 링크 <> 처리

yous commented on source/_posts/2016-12-02-weekly.md:
```diff
@@ -0,0 +1,90 @@
+[NodeSource](https://nodesource.com/) announced [NodeSource Certified Modules™](https://certified.nodesource.com/) to bring security and trust to untrusted, third-party JavaScript. With [NodeSource Certified Modules](https://certified.nodesource.com/), consumers of the npm ecosystem can now rely on NodeSource as a secure, trusted and verifiable source. [Learn more here](https://nodesource.com/blo ...
+-->
+
+[NodeSource](https://nodesource.com/)는 신뢰할 수 없는 써드파티 자바스크립트에 보안 및 신뢰를 제공하기 위해 [NodeSource Certified Modules™](https://certified.nodesource.com/)를 발표했습니다. [NodeSource Certified Modules](https://certified.nodesource.com/)를 사용한 npm 에코 시스템의 사용자는 안전하고 신뢰 및 검증 가능한 소스로 NodeSource에 의존할 수 있습니다. [여기](https://nodesource.com/blog/announcing-nodesource-certified-modules-for-node-js)에서 자세한 내용을 확인할 수 있습니다. 최신 [ ...
```
- 써드파티 → 서드파티
- 에코 시스템 → '생태계'는 어떻습니까?

devjin0617 replied:
수정하였습니다

yous commented on source/_posts/2016-12-02-weekly.md:
```diff
@@ -0,0 +1,90 @@
+The Node.js Foundation announced progress with efforts to make Node.js VM-neutral -- more information on this news can be found on the Node.js Foundation Medium blog. This morning, the Foundation announced it would oversee the Node.js Security Project to further improve stability for enterprises. More information here.
+-->
+
+Node.js 재단은 Node.js를 VM 중립적으로 만들기 위한 노력으로 성과를 발표했습니다. 이 뉴스에 대한 자세한 내용은 Node.js Foundation Medium 블로그에서 확인할 수 있습니다. 금일 오전, 기업의 안정성을 더욱 향상시키기 위해 Node.js 보안 프로젝트를 관리할 것이라고 발표했습니다. 자세한 내용은 여기를 참조하세요.
```
노력으로 성과를 → '노력의 성과를'은 어떻습니까?

devjin0617 replied:
수정하였습니다.

yous commented on source/_posts/2016-12-02-weekly.md:
```diff
@@ -0,0 +1,90 @@
+### Daily Code Coverage Results
+-->
+
+### 일일 커버리지 결과
```
일일 코드 커버리지 결과 https://github.com/nodejs/nodejs-ko/pull/473/files#diff-093ccea2aed8fe4462311e6596daea89R42

yous commented on source/_posts/2016-12-02-weekly.md:
```diff
@@ -0,0 +1,90 @@
+
+* [Node.js용 NodeSource 인증 모듈 발표](https://nodesource.com/blog/announcing-nodesource-certified-modules-for-node-js): 커뮤니티 지원 코드에 대한 신뢰를 얻을 수 있습니다.
+* [Tor-test를 사용하여 Node.js에서 Tor 연결 확인](https://blog.sqreen.io/identify-tor-connections-node-js-tor-test/)
+* [NeedToNode Recap: Node.js에서의 엔드-투-엔드 보안 강화](https://nodesource.com/blog/achieving-end-to-end-security-with-node-js-recap): 몇개의 취약성이 있는 Node.js 모듈, 그리고 우리가 NodeSource N|Solid, Snyk와 함께 Node.js를 이용하여 쉽게 보안을 강화할 수 있는 방법에 대해 자세한 내용을 알아보세요.
```
- NodeToNode → \#NeedToNode
- > 몇개의 취약성이 있는 Node.js 모듈, 그리고 우리가 NodeSource N|Solid, Snyk와 함께 Node.js를 이용하여 쉽게 보안을 강화할 수 있는 방법에 대해 자세한 내용을 알아보세요.

    Node.js 모듈의 취약성 몇 개를 살펴보고, NodeSource N|Solid, Snyk를 사용하여 쉽게 보안을 강화할 수 있는 방법에 대해 알아보세요. https://github.com/nodejs/nodejs-ko/pull/473/files#diff-093ccea2aed8fe4462311e6596daea89R66

yous commented on source/_posts/2016-12-02-weekly.md:
```diff
@@ -0,0 +1,90 @@
+* [Tor-test를 사용하여 Node.js에서 Tor 연결 확인](https://blog.sqreen.io/identify-tor-connections-node-js-tor-test/)
+* [NeedToNode Recap: Node.js에서의 엔드-투-엔드 보안 강화](https://nodesource.com/blog/achieving-end-to-end-security-with-node-js-recap): 몇개의 취약성이 있는 Node.js 모듈, 그리고 우리가 NodeSource N|Solid, Snyk와 함께 Node.js를 이용하여 쉽게 보안을 강화할 수 있는 방법에 대해 자세한 내용을 알아보세요.
+* [네이티브 Node.js 모듈 작성 - 규모있는 Node.js](https://blog.risingstack.com/writing-native-node-js-modules/)
+* [7개의 npm 방법으로 시간낭비를 줄일 수 있습니다](https://nodesource.com/blog/seven-more-npm-tricks-to-knock-your-wombat-socks-off): 우리는 NPM, front-end 등 다른 것들을 이용해 생산성과 속도를 높일 수 있는 다른 멋지고 유용한 방법를 찾아냈습니다.
```
이전 PR을 참고해주세요. https://github.com/nodejs/nodejs-ko/pull/473/files#diff-093ccea2aed8fe4462311e6596daea89R68

devjin0617 replied:
이전 포스트 제가 작업 해놓고, 제대로 참고를 안했었네요. 비교하여 수정진행하였습니다.

taggon commented on source/_posts/2016-12-02-weekly.md:
```diff
@@ -0,0 +1,90 @@
+* [Tor-test를 사용하여 Node.js에서 Tor 연결 확인](https://blog.sqreen.io/identify-tor-connections-node-js-tor-test/)
+* [NeedToNode Recap: Node.js에서의 엔드-투-엔드 보안 강화](https://nodesource.com/blog/achieving-end-to-end-security-with-node-js-recap): 몇개의 취약성이 있는 Node.js 모듈, 그리고 우리가 NodeSource N|Solid, Snyk와 함께 Node.js를 이용하여 쉽게 보안을 강화할 수 있는 방법에 대해 자세한 내용을 알아보세요.
+* [네이티브 Node.js 모듈 작성 - 규모있는 Node.js](https://blog.risingstack.com/writing-native-node-js-modules/)
+* [7개의 npm 방법으로 시간낭비를 줄일 수 있습니다](https://nodesource.com/blog/seven-more-npm-tricks-to-knock-your-wombat-socks-off): 우리는 NPM, front-end 등 다른 것들을 이용해 생산성과 속도를 높일 수 있는 다른 멋지고 유용한 방법를 찾아냈습니다.
```
"시간 낭비를 줄여주는 npm 사용법 7가지"는 어떤가요?

devjin0617 replied:
수정하였습니다. 감사합니다.

devjin0617 committed:
Translate "Weekly Update - December 2nd, 2016" (#471)

- 노력으로 성과를 → '노력의 성과를'
- 써드파티 → 서드파티
- 에코 시스템 → '생태계'
- NodeToNode → #NeedToNode
- 7개의 npm 방법으로 시간낭비를 줄일 수 있습니다 → "시간 낭비를 줄여주는 npm 사용법 7가지"
- 그 외, 이전 PR 참고하여 수정진행
